guides/internal_delivery_support_requests.md

```diff
@@ -0,0 +1,27 @@
+# Internal delivery team support process
+
+## Process purpose
+This outlines how you can request technical support for internal work including projects and Sales activities. 
```

Comment:
I think this should be delivery team rather than technical (e.g. if you want a DM to write a blog post about agile-something-something)